sql/farming.sql

```diff
@@ -1107,3 +1107,189 @@ COMMENT ON COLUMN landuse_area_conditions.date iS 'The datetime alteration of th
 COMMENT ON COLUMN landuse_area_conditions.landuse_area_uuid is 'The foreign key linking to the landuse area table''s UUID.';
 COMMENT ON COLUMN landuse_area_conditions.landuse_area_condition_type_uuid is 'The foreign key linking to the landuse area condition type table''s UUID.';
 
+-- Gates: added by Jeremy Ferris
+CREATE TABLE IF NOT EXISTS gate_type (
+        id SERIAL NOT NULL PRIMARY KEY,
+    uuid UUID UNIQUE NOT NULL DEFAULT gen_random_uuid(),
+        last_update timestamp default now() NOT NULL,
+        last_update_by TEXT,
+        name TEXT UNIQUE NOT NULL, -- primary_type
+    notes TEXT,
+    image TEXT
```

Comment:
@jeremyferris I think it might be nice to indent the code.

Reply:
@charlesmudima The indentation methodology that I used was derived from the code of the past interns. I would suggest that we come up with an indentation methodology or 'convention' and then one of us could go through and update the prior code as well as our own. Once we have a format agreed upon, I do not mind doing the cleanup work on the rest of the older code - I would just record a macro of some sort, I think. I will contact you further on 8-Intern